.github/workflows/ci.yml

```diff
@@ -21,3 +21,23 @@ jobs:
           coverage: xdebug
       - run: composer install
       - run: vendor/bin/phpunit --coverage-text
+
+  PHPStan:
+    name: PHPStan
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: [ 8.1 ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+          tools: phpstan
```

Comment:
I suppose it would, but at the same time, we could also quickly regenerate the baseline as needed. I'm not sure if it's worth keeping track of the PHPStan version across all repositories.